src/router.ts

```diff
@@ -21,18 +23,21 @@ const routes: any[] = [
       { path: '', name: 'overview', component: Overview },
       { path: 'proposals', name: 'proposals', component: Proposals },
       { path: 'settings', name: 'settings', component: Settings },
-      { path: 'treasury', name: 'treasury', component: Treasury }
+      { path: 'treasury', name: 'treasury', component: Treasury },
+      { path: 'create/:key?', name: 'editor', component: Editor },
+      { path: 'proposal/:pid', name: 'proposal', component: Proposal }
     ]
   },
   {
-    path: '/:id/create/:key?',
-    name: 'editor',
-    component: Editor
+    path: '/profile/:id',
+    name: 'user',
+    component: User
   },
-  { path: '/:space/proposal/:id?', name: 'proposal', component: Proposal },
-  { path: '/profile/:id', name: 'user', component: User },
-  { path: '/create', name: 'create', component: Create },
-  { path: '/explore', name: 'explore', component: Explore }
+  {
+    path: '/:pathMatch(.*)*',
```

Comment:
Maybe we can whitelist `:id` in `space` route, so it doesn't math anything that doesn't look like space ID and swallow those.
```
path: '/:id([^:]+:[A-Za-z0-9]+)',
```

Reply:
@j0hnfl0w what do you think?